lms/djangoapps/courseware/interface.py

```diff
@@ -0,0 +1,197 @@
+"""
```

Comment:
Please move the contents of this file into the top of `lms/djangoapps/courseware/user_state_client.py`